JS/Lesson30/savas/my-app/src/components/Ex6/index.jsx

```diff
@@ -0,0 +1,14 @@
+// ## Exercise 6: Prevent Default
+// Design a link inside a component. Implement an event handler that prevents the default action when the link is clicked.
+
+export const Ex6 = () => {
+
+    function handleClick (event) {
+        event.preventDefault();
+        console.log("Button Clicked!")
+    }
+
+    return (
+        <button onClick={handleClick}>Prevent Default</button>
```

Comment:
Should be like this: 
```suggestion
        <a href='https://https://google.com' onClick={handleClick}>Link to Google</a>
```

Reply:
Anna, I actually made this change, but this comment seems unchanged in the section we added. VS Code gives an alert like "Editor is read-only because the file system of the file is read-only". I clicked the Ex6 file, I can change the codes, but your comment doesn't seem there.